docs/releases/4.2.md

```diff
@@ -160,11 +160,14 @@ If these are used within packages or customisations they will need to be updated
 
 | Name                | New (`classname`)                                                                                      | Old (various)                                                                                        |
 | ------------------- | ------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------- |
+| `icon` (see note)   | `{% icon name='spinner' classname='...' %}`                                                            | `{% icon name='spinner class_name='...' %}`                                                          |
 | `dialog_toggle`     | `{% dialog_toggle classname='...' %}`                                                                  | `{% dialog_toggle class_name='...' %}`                                                               |
 | `paginate`          | `{% paginate pages classname="..." %}`                                                                 | `{% paginate pages classnames="..." %}`                                                              |
 | `tab_nav_link`      | `{% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with classname="..." %}`                      | `{% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with classes="..." %}`                      |
 | `side_panel_button` | `{% include 'wagtailadmin/shared/side_panels/includes/side_panel_button.html' with classname="..." %}` | `{% include 'wagtailadmin/shared/side_panels/includes/side_panel_button.html' with classes="..." %}` |
 
+Note that the `icon` template tag will still support `class_name` until Wagtail version 5 but will trigger a deprecation warning.
```

Comment:
@gasman is it worth me updating 4.2 release notes now that we know this won't be deprecated until 6.0 now.

Reply:
Yep, makes sense 👍

Reply:
Done https://github.com/wagtail/wagtail/pull/9998 - opted for generic wording instead of a specific release.